include/cinder/QuasiRand.h

```diff
@@ -0,0 +1,208 @@
+/*
+ Copyright (c) 2020, The Cinder Project: http://libcinder.org
+ All rights reserved.
+
+ This code is intended for use with the Cinder C++ library: http://libcinder.org
+
+ Portions of this code based on the excellent article by Martin Roberts:
+ http://extremelearning.com.au/unreasonable-effectiveness-of-quasirandom-sequences/
+
+ Redistribution and use in source and binary forms, with or without modification, are permitted provided that
+ the following conditions are met:
+
+	* Redistributions of source code must retain the above copyright notice, this list of conditions and
+	the following disclaimer.
+	* Redistributions in binary form must reproduce the above copyright notice, this list of conditions and
+	the following disclaimer in the documentation and/or other materials provided with the distribution.
+
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED
+ WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
+ PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR
+ ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ POSSIBILITY OF SUCH DAMAGE.
+*/
+
+#pragma once
+
+#include "cinder/Export.h"
+#include "cinder/Vector.h"
+
+#include <cstdint>
+#include <random>
+#include <type_traits>
+
+namespace cinder {
+
+template <typename T = float, std::enable_if_t<std::is_floating_point<T>::value, int> = 0>
+class CI_API QuasiRandT {
+  public:
+	QuasiRandT() = default;
+
+	QuasiRandT( uint32_t seed )
+		: mSeed( seed )
+	{
+	}
+
+	//! Resets the quasi-random generator to the specific seed \a seedValue.
+	void seed( uint32_t seedValue ) { mSeed = seedValue; }
+
+	//! Returns a quasi-random float in the range [0.0f,1.0f).
+	T nextFloat()
+	{
+		static T sIrrational = T{ 1 } / phi( 1 );
```

Comment:
All these static `sIrrational` (and A,B,C variations later in the file) variables should be declared `const`.